test/unit/Identifier/IdentifierTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace BetterReflectionTest\Identifier;
+
+use BetterReflection\Identifier\Identifier;
+use BetterReflection\Identifier\IdentifierType;
+
+/**
+ * @covers \BetterReflection\Identifier\Identifier
+ */
+class IdentifierTest extends \PHPUnit_Framework_TestCase
+{
+    public function testValue()
```

Comment:
Consider splitting into separate methods for the separate public APIs being used